src/components/ButtonGroup/ButtonGroup.tsx

```diff
@@ -1,33 +1,68 @@
 import React, { useRef, forwardRef, useCallback, useMemo, useEffect, useState } from "react";
 import PropTypes from "prop-types";
 import cx from "classnames";
-import Button from "../../components/Button/Button";
+import Button from "../Button/Button";
 import usePrevious from "../../hooks/usePrevious";
 import useMergeRefs from "../../hooks/useMergeRefs";
 import { backwardCompatibilityForProperties } from "../../helpers/backwardCompatibilityForProperties";
 import { baseClassName } from "./ButtonGroupConstants";
 import { ButtonWrapper } from "./ButtonWrapper";
 import "./ButtonGroup.scss";
+import VibeComponentProps from "../../types/VibeComponentProps";
+import { BASE_SIZES, SIZES } from "../../constants/sizes";
+import { ButtonType, Size } from "../Button/ButtonConstants";
 
-const ButtonGroup = forwardRef(
+type ButtonGroupOption = {
+  icon?: string | React.FunctionComponent | null;
+  leftIcon?: string | React.FunctionComponent | null;
```

Comment:
Please change type of both icons to `IconType`

Reply:
I totally missed this 😅 
Fixed the type.